lib/lte_link_control/lte_lc.c

```diff
@@ -740,8 +740,19 @@ int lte_lc_normal(void)
 	return 0;
 }
 
+void lte_lc_psm_req_param_set(const char *req_rptau, const char *req_rat)
```

Comment:
Right now this will crash if we pass `NULL`; at least we have to check that the arguments are non-`NULL` here.
Perhaps it would be better to let the buffers hold the default configuration at compile time, instead of being zero.
That way the buffer is never empty, and we avoid the `strlen` check in `lte_lc_psm_req`.

Reply:
I think string lengths should be checked here. Both must be exatly 8 characters long, otherwise the call to `lte_lc_psm_req` will fail